docker-compose.debug.yml

```diff
@@ -9,12 +9,12 @@ services:
     ports:
       - 5678:5678
     volumes:
-      - ./thatkitebot/:/app/thatkitebot
-      - ./data/:/app/data/
-      - ./trains/:/app/data/trains/
-      - ./.git/:/app/.git
-      - ./logs/:/var/log/thatkitebot/
-      - ./config/policy.xml:/etc/ImageMagick-6/policy.xml
+      - ./thatkitebot/:/app/thatkitebot:Z
+      - ./data/:/app/data/:Z
```

Comment:
since this contains the redis data which is shared with another container, shouldn't it use `:z` instead? 
> The z option tells Podman that two containers share the volume content. As a result, Podman labels the content with a shared content label. Shared volume labels allow all containers to read/write content. The Z option tells Podman to label the content with a private unshared label. Only the current container can use a private volume.

(from the [podman github](https://github.com/containers/podman/blob/main/troubleshooting.md#2-cant-use-volume-mount-get-permission-denied))